budgetportal/tests/test_guides_pages.py

```diff
@@ -1,19 +1,25 @@
 from django.test import Client, TestCase
-from budgetportal.models import GuideIndexPage, GuidePage
+from budgetportal.models import GuideIndexPage, GuidePage, CategoryGuide
 
 
 class GuideIndexPageTestCase(TestCase):
     fixtures = ["test-guides-pages"]
 
     def setUp(self):
         self.guide_index_page = GuideIndexPage.objects.get(id=4)
+        self.guide_page = GuidePage.objects.get(id=5)
+        self.category_guides = CategoryGuide.objects.all()
 
     def test_guide_index_page(self):
         """Simple test of template response for guide index page"""
         response = Client().get(self.guide_index_page.url_path)
         self.assertContains(response, self.guide_index_page.title)
         self.assertContains(response, self.guide_index_page.intro)
 
+        for category_guide in self.category_guides:
```

Comment:
please add an assertion that there's at least one category_guide in this collection - otherwise it's easy for someone to break this test by removing the guides from the fixture and they'll never know

Reply:
good one!